feat!: Update flame_audio to AP 1.0.0 (#1724)
flame-engine/flame: doc/flame_audio/audio_pool.md

# AudioPool

> An AudioPool is a provider of AudioPlayers that leaves them pre-loaded with
> local assets to minimize delays.

Reviewer: ```suggestion
An AudioPool is a provider of AudioPlayers that are pre-loaded with
local assets to minimize delays.
```

> A single Audio Pool always plays the same sound, probably a quick sound

Reviewer: ```suggestion
A single AudioPool always plays the same sound, usually a quick sound
```

effect, like a laser shooting from your ship or a jump sound for your
platformer.

The advantage of using Audio Pool is that by configuring a minimum
(starting) size, and a maximum size, the pool will create and preload
some players, and allow them to be re-used many times.

> You can use the helper method `FlameAudio.createPool` to create Audio
> Pool instances using the same global `FlameAudio.audioCache`.

Reviewer: ```suggestion
You can use the helper method `FlameAudio.createPool` to create AudioPool
instances using the same global `FlameAudio.audioCache`.
```
